srcpkgs/gvfs/template

```diff
@@ -1,9 +1,9 @@
 # Template file for 'gvfs'
 pkgname=gvfs
-version=1.38.1
-revision=2
+version=1.40.0
+revision=1
 build_style=meson
-configure_args="-Dsystemduserunitdir=no -Dtmpfilesdir=no -Dlogind=false
+configure_args="-Dsystemduserunitdir=no -Dtmpfilesdir=no -Dlogind=true
```

Comment:
Is that really necessary, what advantage does enabling elogind support bring? gvfs is used by pcmanfm and currently it is possible to run a LXQT desktop without needing elogind.

Reply:
Oh, `xbps-query -RX` didn't show me that and no one in #xbps object to it. I'll disable it, thanks for noticing :)

Reply:
Done